conf/default-cli-options.js

```diff
@@ -28,5 +28,6 @@ module.exports = {
     fix: false,
     allowInlineConfig: true,
     reportUnusedDisableDirectives: void 0,
+    reportUnusedDisableDirectivesSeverity: void 0,
```

Comment:
These are actually default options for the `CLIEngine` class, not the ESLint CLI. Since that class does not have this option, there's no need to set this here.